spec/features/brexit_checker/question_navigation_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "spec_helper"
+
+RSpec.feature "Navigating Brexit Checker questions", type: :feature do
+  scenario "navigate to previous page" do
+    when_i_visit_the_brexit_checker_flow
+    and_i_answer_the_nationality_question
+    then_i_should_see_the_living_question
+
+    when_i_click_on_the_back_link
+    then_i_should_see_the_nationality_question
+    and_my_previous_nationality_choice_is_selected
+  end
+
+  scenario "navigate to previous question with criteria requirements" do
```

Comment:
I'd suggest we only need one scenario here, as there's no difference in the visuals presented from the user's point of view, and it looks like we're covering another potential scenario (clicking back on the first question) in the unit tests. My other worry with these feature tests (including the ones we have already) is that they're already quite brittle, being coupled to the current structure of the questions; for this additional reason, I think we should minimise the number of scenarios where possible. Fortunately, our unit tests look solid for this.

Reply:
Fair point 👍